From d5524a8fd03307219c98e28ed308db185323f5c6 Mon Sep 17 00:00:00 2001
From: Yao Zhao <yao.zhao@windriver.com>
Date: Wed, 16 Jan 2013 16:51:42 -0500
Subject: [PATCH] remove duplicate definition of clock_was_set in hrtimer.c

CQID: WIND00381917

In new kernel, clock_was_set is defined without any condition, so it
will conflict with the clock_was_set under NOTIFY_SETTIME when
CONFIG_HIGH_RES_TIMERS is not defined.
Remove the clock_was_set under NOTIFY_SETTIME, do_notify_timechange
will be called in new clock_was_set still.

Signed-off-by: Yao Zhao <yao.zhao@windriver.com>
---
 kernel/hrtimer.c |   14 --------------
 1 file changed, 14 deletions(-)

diff --git a/kernel/hrtimer.c b/kernel/hrtimer.c
index cfa283c..0ddd9c9 100644
--- a/kernel/hrtimer.c
+++ b/kernel/hrtimer.c
@@ -729,20 +729,6 @@ void clock_was_set_delayed(void)
 
 #else
 
-#ifdef CONFIG_NOTIFY_SETTIME
-/*
- * Clock realtime was set
- *
- * Change the offset of the realtime clock vs. the monotonic
- * clock.  In the absence of high-res timers, that applies only
- * with CONFIG_NOTIFY_SETTIME.
- */
-void clock_was_set(void)
-{
-	/* Notify any userspace apps that were interested in the change */
-	do_notify_timechange();
-}
-#endif
 static inline int hrtimer_hres_active(void) { return 0; }
 static inline int hrtimer_is_hres_enabled(void) { return 0; }
 static inline int hrtimer_switch_to_hres(void) { return 0; }
-- 
1.7.9.7

